src/components/Layout/Header/NavBar/ManageAccountsMenuItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { EditIcon } from '@chakra-ui/icons'
+import { MenuDivider, MenuItem } from '@chakra-ui/react'
+import React, { useCallback } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+import { useModal } from 'hooks/useModal/useModal'
+
+const editIcon = <EditIcon />
+type ManageAccountsMenuItemProps = {
+  onClose?: () => void
```

Comment:
May want to have an optional `onClick` that will be called instead of `handleManageAccountsMenuItemClick`  to keep on honouring the current behavior of demo wallet

Reply:
Done